pytorch_pfn_extras/dataset/tabular/_transform.py

```diff
@@ -29,49 +30,98 @@ def mode(self):
             self.get_examples([0], None)
         return self._mode
 
+    def _find_candidate_transforms(self, key_indices):
+        # TODO: memoize
+
+        # Assume that all the registered transformations are
+        # disjoint on the outputs
+        transforms = []
+        operands = set()
+        # Look for the transforms that produce the
+        # columns specified in the result
+        # to avoid calculating uneeded columns
+        for s, t in self._transforms:
+            ops, res = s
+            res_idx = _utils._as_key_indices(res, self._keys)
```

Comment:
Converting name to index in `__init__` is more efficient, since `res_idx` does not change. (maybe in another PR).

Reply:
done, all is pre-processed